cabal.project.freeze

```diff
@@ -11,7 +11,6 @@ constraints: any.Cabal ==3.6.3.0,
              aeson -cffi +ordered-keymap,
              any.aeson-pretty ==0.8.9,
              aeson-pretty -lib-only,
-             any.alex ==3.2.7.4,
```

Comment:
Curious, why would you like to remove those dependencies?
Maybe we should adapt the scripts that check or re-freeze if we need to remove them.

Reply:
I have already removed these from the booster. they should not be fixed because the nix version of cabal comes with a different version of alex and happy and cant figure out a build plan using these